v2/internal/frontend/desktop/darwin/frontend.go

```diff
@@ -337,7 +336,11 @@ func (f *Frontend) processMessage(message string) {
 }
 
 func (f *Frontend) Callback(message string) {
-	f.ExecJS(`window.wails.Callback(` + strconv.Quote(message) + `);`)
+	escaped, err := json.Marshal(message)
+	if err != nil {
+		panic(err)
```

Comment:
Can we make it not panic here?  That would look like a crash for a user and be really hard to locate.

Reply:
Likewise for the other ones.

Reply:
What would you do though? If your callbacks fail then you have a pretty catastrophic error. I'm trying to think of an alternative

Reply:
As far as I can tell, a panic is appropriate here because `json.Marshal` should never return an error in response to marshaling a string, so if it did that, I would consider that to be unspecified behavior.

On a separate note, I notice that json.Marshall will do a certain amount of html escaping on a string. I can't think how it would be a problem, but worth noticing. See: https://pkg.go.dev/encoding/json#Marshal

> String values encode as JSON strings coerced to valid UTF-8, replacing invalid bytes with the Unicode replacement rune. So that the JSON will be safe to embed inside HTML <script> tags, the string is encoded using HTMLEscape, which replaces "<", ">", "&", U+2028, and U+2029 are escaped to "\u003c","\u003e", "\u0026", "\u2028", and "\u2029". This replacement can be disabled when using an Encoder, by calling SetEscapeHTML(false).